### ISSUE TITLE
Describe hovering and z-order reordering in interactive docs

### ISSUE THREAD

**Issue by joelostblom** [altair-viz/altair#3242]
Include an example of how to bring points to top when hovered as here https://github.com/vega/vega-lite/pull/9136

**Comment by shivam200446** [comment]
I want to work on it. assign me!

**Comment by joelostblom** [comment]
Thank you @shivam200446 , just to clarify what I had in mind when opening this issue. I think we can add a short section to the interactive docs page (not the gallery) that explains how to create a hover ('mouseup'/'pointerup') interaction in a chart and then one more example that demonstrates the z-ordering as in the linked example below. I think that is a good base, but feel free to make suggestions of your own in the PR as well

**Comment by josephburkhart** [comment]
Googling this issue led me here, after I couldn't find an answer on stack overflow. Has this issue been addressed? It seems from [this example](https://altair-viz.github.io/gallery/selection_zorder.html) in the docs that it has.

**Comment by joelostblom** [comment]
Yes, this was closed by https://github.com/vega/altair/pull/3362, thanks for flagging it.
